transitions/registration.js

```diff
@@ -272,5 +252,43 @@ module.exports = {
                 callback();
             }
         });
+    },
+    addPatient: function(options, callback) {
+        var doc = options.doc,
+            db = options.db,
+            audit = options.audit,
+            id = doc.patient_id,
+            patientNameField = (options.params && options.params.length && options.params[0]) || 'patient_name';
```

Comment:
As above, event.params is a string. For passing multiple params (used in other triggers) you can comma separate them. On line 164 we see `options.params = event.params.split(',')`, which means options.params is an array, even if it doesn't contain any commas (as in this trigger). Again, needs a refactor one day...